src/middlewares/auth.middleware.js

```diff
@@ -32,6 +32,16 @@ const authMiddleware = {
       ResponseService.setError(401, 'No Token supplied');
       return ResponseService.send(res);
     }
+  },
+
+  checkUserRole: async (req, res, next) => {
```

Comment:
This function can be renamed to reflect the right role it is checking for. `checkUserRole` can be confused to mean you are checking the role of a `user` and not a `super admin`